Document/0x04f-Testing-Network-Communication.md

```diff
@@ -205,67 +205,123 @@ If you want to verify whether your server supports the right cipher suites, ther
 
-## Intercepting Traffic on the Network Layer
+Once traffic is routed through your host, use tools like Wireshark or tcpdump to inspect and analyze it.
 
-Dynamic analysis by using an interception proxy can be straight forward if standard libraries are used in the app and all communication is done via HTTP. But there are several cases where this is not working:
+## MASTG-TECH: Achieving a MITM Position via ARP Spoofing
 
-- If mobile application development platforms like [Xamarin](https://www.xamarin.com/platform "Xamarin") are used that ignore the system proxy settings;
-- If mobile applications verify if the system proxy is used and refuse to send requests through a proxy;
-- If you want to intercept push notifications, like for example GCM/FCM on Android;
-- If XMPP or other non-HTTP protocols are used.
+When proxy-based interception fails due to non-HTTP protocols or proxy-aware apps, **ARP Spoofing** can be used to redirect network traffic. ARP Spoofing is a **Layer 2 attack** that allows an attacker to impersonate the network gateway, forcing the mobile device to send its traffic through the attacker's machine.
 
-In these cases you need to monitor and analyze the network traffic first in order to decide what to do next. Luckily, there are several options for redirecting and intercepting network communication:
+This technique works against any device and operating system as the attack is executed on OSI Layer 2. When you are MITM, you might not be able to see clear text data, as the data in transit might be encrypted by TLS, but it will give you valuable information about the hosts involved, the protocols used, and the ports the app is communicating with.
 
-- Route the traffic through the host computer. You can set up host computer as the network gateway, e.g. by using the built-in Internet Sharing facilities of your operating system. You can then use @MASTG-TOOL-0081 to sniff any traffic from the mobile device.
-- Sometimes you need to execute a MITM attack to force the mobile device to talk to you. For this scenario you should consider @MASTG-TOOL-0076 or use your own access point to redirect network traffic from the mobile device to your host computer (see below).
-- On a rooted device, you can use hooking or code injection to intercept network-related API calls (e.g. HTTP requests) and dump or even manipulate the arguments of these calls. This eliminates the need to inspect the actual network data. We'll talk in more detail about these techniques in the "Reverse Engineering and Tampering" chapters.
-- On macOS, you can create a "Remote Virtual Interface" for sniffing all traffic on an iOS device. We'll describe this method in the chapter "Basic Security Testing on iOS".
+To execute an ARP Spoofing attack, you can use @MASTG-TOOL-0076 (bettercap) or set up a rogue access point that routes traffic through your host computer.
```

Comment:
```suggestion
To execute an ARP Spoofing attack, you can use @MASTG-TOOL-0076 or set up a rogue access point that routes traffic through your host computer.
```
If you're using a rogue AP, there's no point in ARP spoofing; data is already flowing through your host.